index.html

```diff
@@ -19,7 +19,6 @@
          <nav>
             <i id="id-menu-hamburger" class="menu-hamburg fa fa-bars"></i>
             <ul class="menu">
-               <li class="menu-option">Hello</li>
                <li class="menu-option">Portfolio</li>
                <li class="menu-option">About</li>
                <li class="menu-option">Contact</li>
```

Comment:
- [Optional] On the desktop version you seem to have more spacing between menu items, fixing this will give you even better layout.


### Your design
![your](https://user-images.githubusercontent.com/69077061/193392887-22e5fdec-4933-4347-8668-14d63426e8af.PNG)

### [Figma design  ](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122)
![figma](https://user-images.githubusercontent.com/69077061/193392896-a0af062f-6530-4e7f-a546-9467607429ad.PNG)

Reply:
ok. I do it. Thanks.